src/server.c

```diff
@@ -6053,6 +6053,7 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "mem_replica_full_sync_buffer:%zu\r\n", server.repl_full_sync_buffer.mem_used,
             "mem_asm_migrate_output_buffer:%zu\r\n", mh->asm_migrate_output_buffer,
             "mem_asm_import_input_buffer:%zu\r\n", mh->asm_import_input_buffer,
+            "mem_slot_migration_sync_buffer_peak:%zu\r\n", asmGetPeakSyncBufferSize(),
```

Comment:
maybe `mem_asm_migrate_output_buffer` and `mem_asm_import_input_buffer` should be change to
`mem_slot_migration_output_buffer` and `mem_slot_migration_input_buffer`?